### ISSUE TITLE
Accessibility - ensure that screen reader experience matches what's visually on screen

### ISSUE THREAD

**Issue by nateplusplus** [nateplusplus/pushin#74]
**Is your feature request related to a problem? Please describe.**
This package uses opacity to show and hide layers in a specific timing while scrolling. However, if someone is using a screen reader, any text in those layers will be read straight through, without being timed to the user's scrolling. Since the screen reader will be reading things that are not visible, this could be a confusing experience for users who can see to some degree but also use screen readers.

**Describe the solution you'd like**
Screen readers should read any text in layers when they appear, and should ignore text in layers that are not visible.

**Additional context**
- [Accessible hiding](https://a11y-guidelines.orange.com/en/web/components-examples/accessible-hiding/)
- [Pluralsight](https://www.pluralsight.com/guides/how-to-hide-text-from-screen-readers)

**Comment by nateplusplus** [comment]
This is fixed in v4.1.3
